### PR TITLE
8343170: java/awt/Cursor/JPanelCursorTest/JPanelCursorTest.java does not show the default cursor

### PR DESCRIPTION
**Pull request by DamonGuy**
Change test instructions and drawString to say "Text" instead of "Default" for cursor type. Added instructions to test `setCursor` for the frame as well. Also increased the column count to fit the instructions in linux, as it was mentioned that the instructions didn't fit there.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8343170](https://bugs.openjdk.org/browse/JDK-8343170): java/awt/Cursor/JPanelCursorTest/JPanelCursorTest.java does not show the default cursor (**Bug** - P4)

### Reviewers
 * [Alisen Chung](https://openjdk.org/census#achung) (@alisenchung - Committer) Review applies to [77a7ccfb](https://git.openjdk.org/jdk/pull/22055/files/77a7ccfbd83415f1b6ffe1eb6bc8c8109ce7667d)
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - **Reviewer**)
 * [Alexander Zvegintsev](https://openjdk.org/census#azvegint) (@azvegint - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/22055/head:pull/22055` \
`$ git checkout pull/22055`

Update a local copy of the PR: \
`$ git checkout pull/22055` \
`$ git pull https://git.openjdk.org/jdk.git pull/22055/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 22055`

View PR using the GUI difftool: \
`$ git pr show -t 22055`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/22055.diff">https://git.openjdk.org/jdk/pull/22055.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/22055#issuecomment-2472101672)
</details>

### CODE REVIEW

**Comment by alisenchung** [comment]
I notice one more cursor type (resize cursor?) in between the "move" panel and the panel containing the crosshair button and hand panel. I think it's caused by the JSplitPane resizing option. Since this test is about checking different cursor types while hovering parts of the frame, it might be worth either adding this cursor to the instructions or setting the JSplitPane to not be resizable to remove the cursor.

**Review by alisenchung — COMMENTED** [review]
add frame title, probably add final to instructions field since you have it in all caps

**Comment by DamonGuy** [comment]
> I notice one more cursor type (resize cursor?) in between the "move" panel and the panel containing the crosshair button and hand panel. I think it's caused by the JSplitPane resizing option. Since this test is about checking different cursor types while hovering parts of the frame, it might be worth either adding this cursor to the instructions or setting the JSplitPane to not be resizable to remove the cursor.

I saw this when initially converting the test, but since the original bug description and the test summary is checking for the functionality of `setCursor()`, I don't think it's worth adding a check for the resize cursor.

But I realized that the frame's wait cursor works on all OS's and the frame isn't part of the original test instructions anyway, so I repurposed the wait cursor for the pane instead. Simplifies the instructions too.

**Comment by alisenchung** [comment]
> the frame isn't part of the original test instructions anyway

The original reproducer didn't have test instructions so it's more likely the person who wrote the instructions just forgot about testing the wait cursor. The idea of the original test is that there was a bug where setCursor works for JComponent and JFrame but doesn't work for JPanel. 

So the left side of this test (move cursor) tests for JComponent,
the bottom right (hand) tests for JPanel,
the button (crosshair) tests for "various swing components",
and if you move the cursor outside all of these components onto the JFrame itself, it would test for the JFrame (wait cursor).

The test instructions probably missed the JFrame section of the test since the area on the test where the JFrame is tested is so small, but I think it is worth keeping it. To make the test area bigger you can set the JSplitPane to be added to the "North" of the frame instead of the "Center" and explicitly setSize(300,200) instead of pack(). If you choose to do this you should also add a step in the instructions to test the JFrame setCursor, something like
`4. Verify that the cursor changes to the wait cursor when outside all other components at the bottom of the JFrame`

**Comment by alisenchung** [comment]
If you want to simplify the test instructions between platforms I would rather change the move cursor to the text cursor and leave all the other cursors.

**Review by alisenchung — COMMENTED** [review]
fix spacing error at line 107

File: test/jdk/java/awt/Cursor/JPanelCursorTest/JPanelCursorTest.java [hunk 1]
```diff
@@ -78,11 +80,12 @@ public static JFrame createUI() {
         j.setLeftComponent(pane);
         j.setRightComponent(panel);
         j.setContinuousLayout(true);
-
-        frame.getContentPane().add("Center", j);
-        pane.setCursor(Cursor.getPredefinedCursor(Cursor.MOVE_CURSOR));
+        j.setSize(200, 200);
+
```
Comment (alisenchung) [inline test/jdk/java/awt/Cursor/JPanelCursorTest/JPanelCursorTest.java:84]: looks like theres some whitespaces here

**Review by alisenchung — APPROVED** [review]
looks good once whitespaces removed

**Review by honkar-jdk — APPROVED** [review]
LGTM

**Comment by honkar-jdk** [comment]
@DamonGuy Tested on Windows and it works as expected, does the test instructions hold good for all platforms ?

**Review by azvegint — APPROVED** [review]

**Comment by DamonGuy** [comment]
> @DamonGuy Tested on windows and it works as expected, does the test instructions hold good on all platforms ?

Yup! Changed the cursor types so they're applicable on all platforms. Thanks

**Review by honkar-jdk — APPROVED** [review]

**Pull request closed without merging**